phpunit.xml.dist

```diff
@@ -31,25 +30,18 @@
 
         <!-- Optional modules that aren't in recipes -->
         <testsuite name="Default">
-            <directory>vendor/silverstripe/subsites/tests</directory>
-            <directory>vendor/silverstripe/crontask/tests</directory>
             <directory>vendor/silverstripe/gridfieldqueuedexport/tests</directory>
             <directory>vendor/silverstripe/errorpage/tests</directory>
-            <directory>vendor/silverstripe/registry/tests</directory>
             <directory>vendor/silverstripe/textextraction/tests</directory>
             <directory>vendor/tractorcow/silverstripe-fluent/tests</directory>
         </testsuite>
 
         <testsuite name="recipe-ccl">
```

Comment:
These recipes aren't around anymore. These should be restructured so that either:

a) similar modules are grouped together in test suites, or
b) test suites are grouped to include roughly the same number of modules, or
c) test suites are grouped so that unit tests take roughly the same amount of time per test suite.

e.g. having hybridsessions in its own suite is a waste of a runner.

Reply:
Updated